app/scripts/modules/azure/serverGroup/serverGroup.transformer.js

```diff
@@ -79,10 +77,22 @@ module.exports = angular
 
       if (typeof command.customScriptsSettings !== 'undefined') {
         configuration.customScriptsSettings.commandToExecute = command.customScriptsSettings.commandToExecute;
-        if (Array.isArray(command.customScriptsSettings.fileUris)) {
-          configuration.customScriptsSettings.fileUris = command.customScriptsSettings.fileUris;
-        } else {
-          configuration.customScriptsSettings.fileUris = [command.customScriptsSettings.fileUris];
+        if (
+          typeof command.customScriptsSettings.fileUris !== 'undefined' &&
```

Comment:
You can use lodash's `isEmpty` method here to simplify a bit:

```javascript
if (!_.isEmpty(command.customScriptsSettings.fileUris)) {
  var fileUrisTemp = command.customScriptsSettings.fileUris;
  // ...
}
```

You'll need to add lodash as an import to the top of the file: `const _ = require('lodash');`